src/Sulu/Bundle/AdminBundle/Resources/js/containers/Application/Application.js

```diff
@@ -1,10 +1,11 @@
 // @flow
-import './application.scss';
+import './global.scss';
```

Comment:
This file is not loaded in react-styleguidist, so the examples there are not rendered as in the real usage (one of the reasons I don't like that solution too much at all 😕)